app/components/job-schedule/action/add/patch-job-schedule-form.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component } from "@angular/core";
+import { FormBuilder } from "@angular/forms";
+import { Observable } from "rxjs";
+
+import { autobind } from "@batch-flask/core";
+import { NotificationService } from "@batch-flask/ui/notifications";
+import { SidebarRef } from "@batch-flask/ui/sidebar";
+import { JobScheduleCreateDto, JobSchedulePatchDto } from "app/models/dtos";
+import { JobScheduleService } from "app/services";
+import { JobScheduleCreateBasicDialogComponent } from "./job-schedule-create-basic-dialog.component";
+
+@Component({
+    selector: "bl-patch-job-schedule-form",
+    templateUrl: "job-schedule-create-basic-dialog.html",
+})
+export class PatchJobScheduleComponent extends JobScheduleCreateBasicDialogComponent {
```

Comment:
I added a new functionalities that if you have a title attribute on a form it will use that on the quick access at the top for switching forms

Reply:
you can check, I updated create batch account